src/main/kotlin/org/jitsi/jicofo/JicofoServices.kt

```diff
@@ -94,20 +94,24 @@ open class JicofoServices(
             }
         }
 
-        val focusJid = XmppConfig.client.username.toString() + "@" + XmppConfig.client.domain.toString()
         conferenceRequestHandler = ConferenceRequestHandler(
             focusManager = focusManager,
-            focusAuthJid = focusJid,
+            focusAuthJid = XmppConfig.client.username.toString() + "@" + XmppConfig.client.domain.toString(),
             isFocusAnonymous = StringUtils.isBlank(XmppConfig.client.password),
             authAuthority = authenticationAuthority,
             reservationSystem = reservationSystem
         )
-        focusComponent = FocusComponent(XmppComponentConfig.config, conferenceRequestHandler).apply {
-            val configService = ServiceUtils2.getService(bundleContext, ConfigurationService::class.java)
-            loadConfig(configService, "org.jitsi.jicofo")
-            authenticationAuthority?.let { setAuthAuthority(authenticationAuthority) }
+
+        focusComponent = if (XmppComponentConfig.config.enabled) {
```

Comment:
I forget, is this use of "component" something that we're working on getting rid of?  Or is there something that still needs it?

Reply:
Yes, I'm working on getting rid of it. Jitsi-meet still uses the jid of the component, I think that's the last use.